testing/shunit2/APKBUILD

```
@@ -0,0 +1,27 @@
+# Contributor: Olliver Schinagl <oliver@schinagl.nl>
+# Maintainer: Olliver Schinagl <oliver@schinagl.nl>
+pkgname=shunit2
+pkgver="2.1.7"
+pkgrel=0
+pkgdesc="shUnit2 is a xUnit based unit test framework for Bourne based shell scripts."
+url="https://github.com/kward/shunit2/"
+arch="noarch"
+license="Apache-2.0"
+subpackages="${pkgname}-doc"
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/kward/${pkgname}/archive/v${pkgver}.tar.gz"
+
+check()
+{
+	TMPDIR="$(mktemp -d -p "/tmp/" shunit2.XXXXXXXX)" ./gen_test_report.sh
```

```suggestion
	local tempdir
	tempdir="$(mkdir -p "$TMPDIR"; mktemp -d -t shunit2.XXXXXXXX)"
	TMPDIR="$tempdir" ./gen_test_report.sh
```

TMPDIR could already be set (even to a non-existent path), so we shouldn't hard-code `/tmp` here.

---

I'd rather not use local, as that is a bashism specific thing (future proofing). However I fully agree that TMPDIR could be set, and we should only set it if it is not set.